Resources/config/css.yml

```diff
@@ -90,6 +90,8 @@ system:
                 - 'bundles/ezplatformui/css/views/fields/edit/richtext.css'
                 - 'bundles/ezplatformui/css/views/fields/edit/isbn.css'
                 - 'bundles/ezplatformui/css/views/fields/edit/user.css'
+                - 'bundles/ezplatformui/css/views/dashboard/dashboard.css'
```

Comment:
same remark as in https://github.com/ezsystems/PlatformUIBundle/pull/584/files#r62842220, the path of the view and its stylesheets (and the tests) should be consistent. So no `dashboard/` level here.